modules/README.md

```diff
@@ -4,113 +4,114 @@ In this section you can find useful information regarding the Modules that are c
 
 | Name | Provider namespace | Resource Type |
 | - | - | - |
-| [Azure Active Directory Domain Services](https://github.com/Azure/ResourceModules/tree/main/modules/Microsoft.AAD/DomainServices) | `MS.AAD` | [DomainServices](https://github.com/Azure/ResourceModules/tree/main/modules/Microsoft.AAD/DomainServices) |
```

Comment:
Same comment as for the root `README.md` - please undo these changes which are automatically happening if a module is changed in main.